packages/core/src/Toast/ToastsProvider.tsx

```diff
@@ -11,8 +11,8 @@ import { useToastCallbacks, SimpleToastsDurations } from './useToasts'
 import { ToastAction, NI, ToastsContext } from './context'
 
 export interface ToastsPlacement {
-  readonly justify: 'center' | 'right'
-  readonly top: string
+  readonly justify?: 'center' | 'right'
+  readonly top?: string
```

Comment:
Either remove the optionals or make sure it is merged correctly. The spread above only merges it on one level.

Reply:
Done